Helper/Cart.php

```diff
@@ -367,13 +390,37 @@ public function getCartData($payment_only, $place_order_payload)
 
 		// add discount data
 		$cart['discounts'] = $shippingAddress->getDiscountAmount() ? [[
-			'description' => 'Discount',
-			'amount'      => -$this->getRoundAmount($shippingAddress->getDiscountAmount())
-		]] : null;
+			'description' => __('Discount ') . $shippingAddress->getDiscountDescription(),
+			'amount'      => abs($this->getRoundAmount($shippingAddress->getDiscountAmount()))
+		]] : [];
+
+		$totals = $quote->getTotals();
+
+		//$this->logHelper->addInfoLog(var_export($totals, 1));
+
+		foreach ($this->discount_types as $discount) {
+
+			if (@$totals[$discount] && @$totals[$discount]->getValue()) {
+
+				$cart['discounts'][] = [
+					'description' => @$totals[$discount]->getTitle(),
+					'amount'      => abs($this->getRoundAmount($totals[$discount]->getValue()))
+				];
+			}
+		}
+
+		if (@$quote->getCustomerBalanceAmountUsed()) {
+			$cart['discounts'][] = [
+				'description' => __('1% Store Credit', $quote->getCustomerBalanceAmountUsed()),

```

Comment:
what is this `'1% Store Credit'` ?

Reply:
It a display name for Customer Balance, a form of in-house credit / points payment option that we see as a discount. It used to figure as a third party plugin of several vendors / implementations, but is now officially incorporated in the cloud version of the Magento 2 Enterprise Edition.

Reply:
Apologies for bypassing you on the merge. Will not do in the future.